frontend/views/containers/access/LoginForm.vue

```diff
@@ -74,6 +74,10 @@ export default ({
   },
   methods: {
     async login () {
+      if (typeof document.body.dispatchEvent === 'function') {
+        this.$refs.username.dispatchEvent(new Event('input'))
+        // TODO: password input too
+      }
```

Comment:
Hmm, what is the purpose of this code?

This looks like it either needs a comment explaining what it's doing, or needs to be removed. I'm noticing a weird console error now whenever I bring up the LoginModal and dismiss it while the username field is empty:

https://github.com/okTurtles/group-income/assets/138706/e090684f-fafa-44b6-8280-371e281f87dd

Reply:
> Hmm, what is the purpose of this code?

It's for making `Input` event manually when user click `Login` button so that the `this.form.username` represents the current value of input element even though the injected script updated its value. If all the chrome extension emit this `Input` event after they update the value of input element, this isn't needed.
I'll remove this because `LastPass` extension do emit that event and let us have the correct username.